helpers/money.js

```diff
@@ -17,19 +15,32 @@ function numberFormat(value, n, s, c) {
 }
 
 const factory = globals => {
-    return function(value) {
+    return function(...args) {
         const siteSettings = globals.getSiteSettings();
         const money = siteSettings.money;
 
-        if (!_.isNumber(value)) {
-            return '';
+        // remove options hash object
+        args.pop();
```

Comment:
Are we able to exclude this by doing something like `return function(options, ...args)` to pull out the first param as "options" and just use the remaining in `args` to avoid this step?

Reply:
not really, cause I'm removing the last parameter, rather then the first one. 
So it should be like `return function(...args, options)`, but this kind of syntax is not supported 😒

Reply:
@jmwiese the thing is that number of arguments may vary. Just to be sure, that hash object (that has handlebars utility functions) was removed, I'm using a `pop()`. We are also using same approach in other helpers, to actually read that hash object by assigning it to a variable.